src/actions/users/update.ts

```diff
@@ -27,6 +27,8 @@ export async function updateUser(id: number, rawdata: FormData | UpdateUserSchem
     }
 }
 
+// These function should maybe be in another place than server actions? @Paulijuz
```

Comment:
this comment should be removed :)

Reply:
This is still a problem, will be fixed in another PR